### PR TITLE
(MODULES-8584) Handle multiple escaped quotes in comments properly

### PR DESCRIPTION
**Pull request by mateusz-gozdek-sociomantic**

### CODE REVIEW

File: spec/fixtures/iptables/conversion_hash.rb [hunk 1]
```diff
@@ -255,10 +255,10 @@
     },
   },
   'string_escape_sequences' => {
-    line: '-A INPUT -m comment --comment "000 parse escaped \\"s, \\\'s, and \\\\s"',
+    line: '-A INPUT -m comment --comment "000 parse escaped \\"s, \\"s, \\\'s, \\\'s, \\\\s and \\\\s"',
     table: 'filter',
     params: {
-      name: '000 parse escaped "s, \'s, and \\s',
+      name: '000 parse escaped "s, "s, \'s, \'s, \\s and \\s',
```
Comment (david22swan) [inline spec/fixtures/iptables/conversion_hash.rb]: Why did you double up each of these?

**Comment by david22swan** [comment]
@mateusz-gozdek-sociomantic One comment, but otherwise just needs some test coverage added and I think I would be happy to merge.

File: spec/fixtures/iptables/conversion_hash.rb (hunk 1 above)
Comment (david22swan) [reply spec/fixtures/iptables/conversion_hash.rb]: Ah, I'm an idiot. Need to stop reviewing first thing on a monday. Will make an adhoc run, and if it passes will merge.

Comment (mateusz-gozdek-sociomantic) [reply spec/fixtures/iptables/conversion_hash.rb]: Haha, cheers!

**Comment by david22swan** [comment]
screen shot 2019-02-05 at 11 03 09 am
Sorry for the delay.
Anyway looks good to me

**Comment by david22swan** [comment]
@mateusz-gozdek-sociomantic Thanks for putting in the work :)

**Comment by mateusz-gozdek-sociomantic** [comment]
Thanks for quick feedback!

**Pull request merged**